mod_form.php

```diff
@@ -227,9 +227,10 @@ public function definition() {
 
             foreach ($users as $user) {
                 $certid = null;
+                $useremail = strtolower($user->email);
```

Comment:
Same here - I wouldn't mind not defining the variable as `if ($certificate->recipient->email == strtolower($user->email))`.